kubernetes/structures.go

```diff
@@ -141,6 +152,14 @@ func isInternalKey(annotationKey string) bool {
 	return false
 }
 
+func byteMapToBase64Map(m map[string][]byte) map[string]string {
```

Comment:
@alexsomesan I believe this function name and the one below could have been prefixed by `flatteners` given how they are used. However, they do not have specific source/target type domain ties.

WDYT?

Reply:
That sounds like a fair assumption. Go ahead and rename if you so prefer.

Reply:
Functions renamed.